examples/README.Rmd

```diff
@@ -142,16 +144,6 @@ package.
 print_yaml("pr-commands.yaml")
 ```
 
-## Render Rmarkdown
-
-`usethis::use_github_action("render-rmarkdown")`
-
-This example automatically re-builds any Rmarkdown file in the repository whenever it changes and commits the results to the same branch.
-
-```{r echo = FALSE, results = "asis"}
-print_yaml("render-rmarkdown.yaml")
-```
-
```

Comment:
Can you please undo moving sections around in the README? Just inserting the new section should be fine. Thanks!

Reply:
Done!